src/main/java/itm/oss/splitter/Exporter.java

```diff
@@ -1,12 +1,40 @@
 package itm.oss.splitter;
 
 import java.io.*;
+import java.math.RoundingMode;
 import java.util.ArrayList;
 
 public class Exporter {
 
   public static void writePaymentsCsv(String path, ArrayList<Payment> pays) throws IOException {
     // TODO (Issue 9): write CSV with header: from,to,amount
-    throw new UnsupportedOperationException("writePaymentsCsv() not implemented yet");
+
+    File file = new File(path);
+
+    // Automatically close using try-with-resources
+    try (FileWriter writer = new FileWriter(file)) {
```

Comment:
`try (FileWriter writer = new FileWriter(file))` will fail if the parent directory of path does not exist. `FileWriter` does not create directories and will throw an exception (e.g., `FileNotFoundException`). I also couldn’t find any test or verification that covers this case.

The below is the error that I faced when I run the test after deleting `data/` repository.
<img width="1445" height="309" alt="image" src="https://github.com/user-attachments/assets/8867f7e7-0f84-48e3-9e70-354d013c72fc" /> 

```suggestion
    +// Ensure parent directory exists to avoid FileNotFoundException
+File parent = file.getParentFile();
+if (parent != null && !parent.exists()) { // Check if there is no parent direcotry
+    java.nio.file.Files.createDirectories(parent.toPath());
+}

    try (FileWriter writer = new FileWriter(file)) {
```

It might be safer to check and create (if necessary) the parent directory before opening the writer so that the export works consistently across environments. :+1:

Reply:
Applied your suggestion. We added directory existence check before opening FileWriter. Verified that export now works even if data/ folder is missing. Thanks for the catch!

Reply:
LGTM, thanks for adding comments, now we can see the process clearly!